src/bika/lims/browser/analysisrequest/templates/analysisrequest_results_interpretation.pt

```diff
@@ -51,7 +51,36 @@
     <input type="hidden" name="submitted" value="1"/>
     <input tal:replace="structure context/@@authenticator/authenticator"/>
 
-    <div class="arresultsinterpretation-container"
+    <!-- Interpretation templates selector -->
+    <div class="row">
+      <div class="col-sm-12 mb-2">
+        <div class="form form-inline">
+          <div class="form-group arinterpretationtemplates-selector">
+              <label class="field mr-2"
+                     for="interpretationtemplate"
+                     i18n:translate="">Templates</label>
+
+              <select id="interpretationtemplate"
+                      name="interpretationtemplate"
+                      class="field form-control mr-2">
+                <option value="" i18n:translate="">Select template</option>
+                <tal:itemplates repeat="template view/get_interpretation_templates">
+                  <option tal:attributes="value template/uid"
+                          tal:content="template/title"/>
+                </tal:itemplates>
+              </select>
+
+              <button id="interpretationtemplate-insert"
+                      name="interpretationtemplate-insert"
+                      type="submit"
+                      class="btn allowMultiSubmit btn-outline-primary btn-sm"
```

Comment:
I think this `allowMultiSubmit` CSS class can be dropped or?

Reply:
Yes, I think so